exercises/rest-api/RestApi.fs

```diff
@@ -0,0 +1,3 @@
+﻿module RestApi
+
+// TODO: implement this module
```

Comment:
At this difficulty level (5), we usually give users a stub implementation. However, in this case I'm not sure we should do that. @robkeim what do you think?

Reply:
@ErikSchierboom what's your rationale behind not wanting to give a stub implementation for this exercise out of curiosity?

Reply:
Well, I'm not completely sure. It was more of a random blurb, so maybe we should create a stub file?

Reply:
A stub file makes the most sense to me in terms of consistency.

Reply:
Agreed. @davidelettieri could you create a stub in the implementation file?

Reply:
I'll do it as soon as I get back from vacation